tests/conftest.py

```diff
@@ -28,6 +29,9 @@ def __init__(self, name, parent, path):
         self.path = path
 
     def runtest(self):
+        readme_path = os.path.join(self.path.dirname, 'README.rst')
```

Comment:
though the readme *will* need to actually be in rST syntax. We're using them to generate some documentation from these tests (for now anyway)

http://repo2docker.readthedocs.io/en/latest/